README.md

```diff
@@ -71,7 +71,7 @@ Radarr is currently undergoing rapid development and pull requests are actively
 * All indexers supported by Sonarr also supported
 * New PassThePopcorn Indexer
 * QBittorrent, Deluge, rTorrent, Transmission and uTorrent download client (Other clients are coming)
-* New TorrentPotato Indexer (Works well with [Jackett](https://github.com/Jackett/Jackett))
+* New Torznab Indexer (Works well with [Jackett](https://github.com/Jackett/Jackett))
```

Comment:
I would leave New TorrentPotato indexer (since Torznab is not new for people coming from Sonarr) and just remove the works well with Jackett part. Then add another point: * Well working Jackett integration via a Torznab Indexer.